src/styles/main.scss

```diff
@@ -1 +1,76 @@
-// put code here
+body {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.wrapper {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+
+  &__hand {
+    transform-origin: bottom center;
+    position: absolute;
+  }
+
+  &__hand--minute {
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+    animation: minute 3600s infinite steps(60);
+  }
+
+  &__hand--second {
+    width: 1.5vmin;
+    height: 38vmin;
+    background-color: #2c8000;
+    animation: second 60s infinite linear;
+  }
+
+  &__center {
+    position: absolute;
+    display: flex;
+    width: 5vmin;
+    height: 5vmin;
+    background-color: #f6a603;
+    border-radius: 50%;
+  }
+}
+
+@keyframes minute {
+  0% {
+    transform: translate(0, -50%) rotate(0deg);
+  }
+  100% {
+    transform: translate(0, -50%) rotate(360deg);
+  }
+}
+
+@keyframes second {
+  0% {
+    transform: translate(0, -50%) rotate(0deg);
+  }
+  100% {
+    transform: translate(0, -50%) rotate(360deg);
+  }
+}
```

Comment:
these keyframes are the same - remove one of them (and rename another)

Reply:
![Screenshot_2](https://github.com/mate-academy/layout_stop-watch/assets/149514552/0505bf09-adf1-4b21-b597-294bcb54c015)